portable/GCC/ARM7_AT91SAM7S/lib_AT91SAM7X256.h

```diff
@@ -1,1469 +1,1396 @@
-        unsigned int flag) // \arg  pio output write to be enabled
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_IsAB_RegisterSet */
+/** \brief Test if PIO AB Register is Set */
+/**---------------------------------------------------------------------------- */
+    __inline int AT91F_PIO_IsAB_RegisterSet( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                             unsigned int flag ) /* \arg  flag to be tested */
+    {
+        return( AT91F_PIO_Get_AB_RegisterStatus( pPio ) & flag );
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_OutputWriteEnable */
+/** \brief Output Write Enable PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_OutputWriteEnable( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                               unsigned int flag ) /* \arg  pio output write to be enabled */
+    {
         pPio->PIO_OWER = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_OutputWriteDisable
-//* \brief Output Write Disable PIO
-//*----------------------------------------------------------------------------
-__inline void AT91F_PIO_OutputWriteDisable(
-        AT91PS_PIO pPio,   // \arg  pointer to a PIO controller
-        unsigned int flag) // \arg  pio output write to be disabled
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_OutputWriteDisable */
+/** \brief Output Write Disable PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_OutputWriteDisable( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                                unsigned int flag ) /* \arg  pio output write to be disabled */
+    {
         pPio->PIO_OWDR = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_GetOutputWriteStatus
-//* \brief Return PIO Output Write Status
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_PIO_GetOutputWriteStatus( // \return PIO Output Write Status
-        AT91PS_PIO pPio) // \arg  pointer to a PIO controller
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_GetOutputWriteStatus */
+/** \brief Return PIO Output Write Status */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PIO_GetOutputWriteStatus( /* \return PIO Output Write Status */
+        AT91PS_PIO pPio )                                 /* \arg  pointer to a PIO controller */
```

Comment:
Does this need to be spread across multiple lines?

Reply:
Answered [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768#discussion_r1313523527)